backend/akr/src/test/java/fi/oph/akr/service/PublicTranslatorServiceTest.java

```diff
@@ -137,7 +145,10 @@ public void listShouldReturnDistinctTowns() {
 
     final PublicTranslatorResponseDTO responseDTO = publicTranslatorService.listTranslators();
 
-    assertEquals(List.of("Kaupunki1", "Kaupunki2"), responseDTO.towns());
+    assertEquals(
+      List.of(createTownAndCountryDTO("Kaupunki1"), createTownAndCountryDTO("Kaupunki2")),
```

Comment:
Tässä `listShouldReturnDistinctTowns` olisi hyvä testata myös sellainen keissi, että meillä on esim. kääntäjät, joilla kaupunki-maa parit (town: "Kaupunki1", country: null) ja (town: "Kaupunki1", country: "SE") tms. ja katsoa että molemmat palautetaan jos tarkoitus näin olikin. Myös halutessa keissi jossa country: "FIN" voi testata.